### DISCUSSION TITLE
Support `position` parameter for nnn.nvim floating window

### DISCUSSION THREAD

**Discussion by allumik** [mcchrish/nnn.vim#120] [Ideas]
Currently it seems that the `position` parameter for the floating window does not actually do anything. For example with configuration:

```lua
require("nnn").setup({
  set_default_mappings = 0,
  replace_netrw = 1,
  session = "local",
  layout = {
    window = { width = 1, height = 0.4, position = 'bottom' }
  }
})
```
The terminal is still centered not in the bottom as one would expect.

**Comment by mcchrish** [comment]
`yoffset` and `xoffset` are the options for this. If you want to place at the bottom, set `yoffset` to 1.

**Comment by allumik** [comment]
It works! I forgot the `offset` parameters.
